src/modules/status.rs

```diff
@@ -54,12 +54,17 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
+    let segment_format = match config.pipestatus_segment_format {
+        None => config.format,
+        Some(format) => format,
+    };
```

Comment:
This is much simpler:
```suggestion
    let segment_format = config.pipestatus_segment_format.unwrap_or(config.format);
```